app/components/workflow-files.js

```diff
@@ -45,6 +45,10 @@ export default Component.extend({
           for (let i = 0; i < uploads.files.length; i++) {
             const file = uploads.files[i];
             if (file) {
+              if (file.size > (1024 * 1024 * 100)) {
+                toastr.error(`Your file '${file.name}' is bigger than 100MB (${Number.parseFloat(file.size / 1024 / 1024).toPrecision(3)}MB) and was not added to the submission.`);
+                continue;
```

Comment:
I found the current file size being in parentheses after the limit slightly confusing :
"Your file 'testfile3.txt' is 113MB. This is larger than the maximum upload size of 100MB and the file was not added to the submission."